lib/lrama/states/item.rb

```diff
@@ -114,7 +114,7 @@ def display_rest
 
       # @rbs (States::Item other_item) -> bool
       def predecessor_item_of?(other_item)
-        rule == other_item.rule && position == other_item.position - 1
+        position == other_item.position - 1 && rule == other_item.rule
```

Comment:
Just clarification, is there any major performance difference between checking `position` first and `rule` first?

Reply:
Oops. This was a trial and error. It didn't seem to affect the speed when I put it back.